src/components/inputs/password-input.tsx

```diff
@@ -0,0 +1,56 @@
+import { Icon } from 'native-base';
+import React, { useState } from 'react';
+import { View, TouchableOpacity, Text } from 'react-native';
+import MaterialIcons from 'react-native-vector-icons/MaterialIcons';
+
+import Input, { InputProps } from './input';
+import { usePasswordInputStyles } from './input.styles';
+
+interface PasswordInputProps extends Omit<InputProps, 'isPassword'> {}
```

Comment:
I dont think `InputProps` has / should have `isPassword` property. This seems redudant.